docs/01-getting-started/02-project-structure.mdx

```diff
@@ -33,7 +33,8 @@ This page provides an overview of the file and folder structure of a Next.js pro
 | `next-env.d.ts`                                                                             | TypeScript declaration file for Next.js |
 | `tsconfig.json`                                                                             | Configuration file for TypeScript       |
 | `jsconfig.json`                                                                             | Configuration file for JavaScript       |
-| `postcss.config.js`                                                                         | Configuration file for Tailwind CSS     |
+| `tailwind.config.js`                                                                        | Configuration file for Tailwind CSS     |
```

Comment:
I actually think both of these should be removed, since they're not part of standard Next.js. Could you remove tailwind and postcss?